spyder/widgets/projects/explorer.py

```diff
@@ -30,6 +30,8 @@
 
 class ExplorerTreeWidget(FilteredDirView):
     """Explorer tree widget"""
+    
+    delete_project = Signal()

```

Comment:
I forgot to tell you that we decided to add the `sig_` prefix for new signals, so this one should be `sig_delete_project` :-)